be/db/dev-db/docker-compose.yaml

```diff
@@ -0,0 +1,16 @@
+version: '3.1'
+services:
+  db:
+    image: postgres:latest
+    restart: always
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+    ports:
+      - 5432:5432
```

Comment:
Please change this to some other random number. I don't feel like solving conflicting port problems and it will be easier to test if our db is running already vs some random one.

Reply:
Also this is for development only. There is no need to keep ports open for production. Also this won't affect how application connects as inside docker services are on the same network. This is only if you want to connect to db via db manager on your host machine.

Reply:
> Please change this to some other random number.

5432 is the default postgres port. Please suggest another port number if you really want it to be changed.

Reply:
why not 7842

Reply:
> why not 7842

if we are suggesting then I am throwing my hat down for 1337 for obvious reasons. (not that it matters) Anything beyond 10k is free world and is not bound to collide with anything important. so either pick one suggested or use random nr generator or just do +1 or -1 and be done with it.

Reply:
Adminer does not ask for port, it assumes port 5432 for postgres. Changing host port does not affect adminer, but it does affect database managers on host machine. Changing the port will only confuse developers. I belive we should keep the default postgres port of '5432'.

Reply:
it does not matter for adminer as adminer is functioning within docker network. Port is only for developers who want to use external db managers

Reply:
Port '5432' will stay for now. This can be changed later if needed.

Reply:
it is needed as @Orzelius requested. forwarded ports on db are useful in development only. And developers tend to have multiple projects and instances running of same application. Fact that you don't have anything running on those ports doesn't mean other could not have like local postgres installations or containers. Changing the port that the service port is forwarded to will be to help out devs with those issues and its a hassle to keep changing that one line if you think to claim that who needs it changed will change it. like I said it will not affect adminer in any way.  just change it to 54321:5432

Reply:
this is still needed

Reply:
Commit 49de9b596bc7854fb673fea4a14129fa0b034c9d changes postgres port from default of '5432' to '7842' as requested by @Orzelius above.